### ISSUE TITLE
Stock Firefox failes to download NXM links on Ubuntu 22.04

### ISSUE THREAD

**Issue by Ratione** [rockerbacon/modorganizer2-linux-installer#382]
The default firefox install has issues executing the NXM handler links. Post-Install Opera does not have this problem. Might shed light on the actual underlying issue here.

**Comment by Flightkick** [comment]
I think this might be the same issue that I am experiencing.

Firefox does not seem to recognize the handler for me.
![Firefox shows empty handler list](https://user-images.githubusercontent.com/8693699/189503965-527073dd-61e8-4775-8e3e-2455baf5ed19.png)

```shell
$ xdg-mime query default x-scheme-handler/nxm
modorganizer2-nxm-handler.desktop
```

Firefox is installed through Flatpak.
Also tested with Chromium based browser which seems to handle the nxm links without issues.

--------

To debug this behavior I tried manually changing `modorganizer2-nxm-handler.desktop` to:
```desktop
[Desktop Entry]
Categories=Game;
Comment[en_US]=
Comment=
Exec=bash -c 'echo "$@" >> /home/xxxxxxxx/.local/share/modorganizer2/input.log'
GenericName[en_US]=
GenericName=
MimeType=x-scheme-handler/nxm;
Name[en_US]=Mod Organizer 2 NXM Handler
Name=Mod Organizer 2 NXM Handler
NoDisplay=true
Path=
StartupNotify=true
Terminal=false
TerminalOptions=
Type=Application
X-DBUS-ServiceName=
X-DBUS-StartupType=
X-KDE-SubstituteUID=false
X-KDE-Username=
```

To see if Firefox passed different arguments to the handler.
I executed `update-desktop-database ~/.local/share/applications/`
And clicked the nxm link again. Firefox opened protontricks but I was presented with a modal 'Select Steam app'.
So the `MimeType=x-scheme-handler/nxm;` does seem to get picked up by Firefox, strange enough it opened protontricks this time while I expected it to not do anything except echo the parameters to the log file as per the `Exec` option. There was no log file present.

I can't debug this any further now but I hope this might help someone else.
